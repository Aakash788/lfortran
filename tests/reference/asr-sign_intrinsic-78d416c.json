{
    "basename": "asr-sign_intrinsic-78d416c",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/sign_intrinsic.f90",
    "infile_hash": "009e10a70f82920141c580c91ea254d2876e5a88114cadb044f7eff3",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-sign_intrinsic-78d416c.stderr",
    "stderr_hash": "e2bb7848b2e1c8feed002d57a8ab30dff63a9deb789751cfd3a2c31d",
    "returncode": 2
}