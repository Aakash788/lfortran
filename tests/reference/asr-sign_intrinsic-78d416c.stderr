semantic error: Type mismatch in statement function, the types must be compatible, type mismatch.
 --> tests/errors/sign_intrinsic.f90:6:15
  |
6 |     signval = sign(x, y)
  |               ^^^^^^^^^^ 
